### PR TITLE
Checkpoint-00

### PR DESCRIPTION
**Pull request by Costle784**

### CODE REVIEW

**Comment by Costle784** [comment]
just noticed I forgot to name the function in question 6. should be
function thisIsAFunction(arr) {
     return arr[0];
}

File: readme.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ For the following quiz, there will be a series of questions or directions follow
 Create a variable and store the string "pizza" in it
 
 ```js
-answer goes here
+var x = 'pizza';
```
Comment (juancgarcia) [inline readme.md:14]: 👍

File: readme.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ var clown = {
 Access the value `"Jonas"` out of the `names` array:
 
 ```js
-answer goes here
+names[0];
```
Comment (juancgarcia) [inline readme.md:42]: 👍

File: readme.md [hunk 3]
```diff
@@ -48,7 +48,9 @@ answer goes here
 Write a for loop such that you say hello(using `console.log`) to each name in the `names` array
 
 ```js
-answer goes here
+for (var i = 0;i < names.length;i++) {
```
Comment (juancgarcia) [inline readme.md:51]: 👍

File: readme.md [hunk 4]
```diff
@@ -59,7 +61,7 @@ answer goes here
 Access the value `"Alfred"` out of the `clown` object
 
 ```js
-answer goes here
+clown.enemy.minions[1];
```
Comment (juancgarcia) [inline readme.md:64]: 👍

File: readme.md [hunk 5]
```diff
@@ -68,15 +70,17 @@ answer goes here
 Set a new property on the object stored in the variable clown. Make it anything you want!
 
 ```js
-answer goes here
+clown.hungry = 'YES';
```
Comment (juancgarcia) [inline readme.md:73]: 👍

File: readme.md [hunk 6]
```diff
@@ -68,15 +70,17 @@ answer goes here
...
 ```
 
 ---
 ### Question 6
 Write a function that takes an array as an argument and returns the array's first value
 
 ```js
-answer goes here
+var firstValue = function(arr) {
```
Comment (juancgarcia) [inline readme.md:81]: 👍

File: readme.md [hunk 7]
```diff
@@ -87,7 +91,7 @@ answer goes here
 What is Git, what problem does it solve? What is the difference between `git` and github?
 
 ```
-answer goes here
+Git is a way to manage version control while working on a program. It documents each change made to the program and makes them easily accessible. This prevents one from having to juggle many different versions of the same file.  Github is a web hosting service that remotely hosts your git repositories.
```
Comment (juancgarcia) [inline readme.md:94]: 👍

File: readme.md [hunk 8]
```diff
@@ -98,7 +102,7 @@ answer goes here
 What is the difference between a fork and a clone?
 
 ```
-answer goes here
+a fork copies a version of someone else's remote repository to one you can edit separately yourself on github. A clone makes a copy of a remote repository to your computer.
```
Comment (juancgarcia) [inline readme.md:105]: 👍

File: readme.md [hunk 9]
```diff
@@ -113,7 +117,7 @@ answer goes here
 How would you link a css file entitled, `hardstyle.css` in an html file?
 
 ```
-answer goes here
+<link rel='stylesheet' type='text/css' href='hardstyle.css'>
```
Comment (juancgarcia) [inline readme.md:120]: 👍

File: readme.md [hunk 10]
```diff
@@ -142,7 +146,9 @@ Write a CSS selector-rule that will select and apply styling to an element with
 
 
 ```
-answer goes here
+#dog-resume {
```
Comment (juancgarcia) [inline readme.md:149]: 👍

File: readme.md [hunk 11]
```diff
@@ -152,5 +158,7 @@ answer goes here
 Write a CSS selector-rule that will select and apply styling to every `<li>` inside of a `<ul>`:
 
 ```
-answer goes here
+li {
```
Comment (juancgarcia) [inline readme.md:161]: So close!

This selector will get all the `li` elements, but if we change the html to include `ol` lists, those will get the style as well.

A more specific selector would be `ul li`
The left side matches the larger encompassing elements (`ul`) and getting more specific towards the right side (`li`)

**Review by juancgarcia — COMMENTED** [review]
10/11 Great Job!

I think you had a good grasp of the pre-work materials. Stay on target with your studying and practice.

**Pull request closed without merging**